components/codeChecker/issueHint/NotUsingExternalResources.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <p class="text-base">
+    {{ $t('codeChecker.notUsingExternalResourcesMeans') }}
+  </p>
+  <!--  -->
+  <CodeCheckerIssueHintCodeBlock class="mt-5" :code="code" lang="html" />
+</template>
+
+<script setup lang="ts">
+const code = `
+  <!-- DON'T ❌ -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.2/p5.js" />
+
+  <!-- DO ✅-->
+  <script src="p5.js" />
```

Comment:
[incorrect](https://github.com/vikiival/kodahash/blob/main/public/index.html)

```suggestion
  <script src="./p5.min.js">
```

Reply:
fixed